### PR TITLE
feat(support): add issueTypes & serviceTypes routes

### PR DESCRIPTION
**Pull request by frenautvh**

### CODE REVIEW

**Comment by FredericEspiau** [comment]
Capture d’écran 2019-09-04 à 11 35 37

**Comment by ovh-cds** [comment]
CDS Report build#1219.0 ✘  
* 
  * build ✘  
  * test ✘

File: src/api/support/support.v6.service.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ angular.module('ovh-api-services').service('OvhApiSupportV6', ($resource, $cache
     create: {
       interceptor,
       method: 'POST',
-      url: '/support/tickets/:id/create',
+      url: '/support/tickets',
```
Comment (FredericEspiau) [inline src/api/support/support.v6.service.js]: We should keep the previous path as well as "One shot ticketing" may not be prodded in the US :)

Comment (frenautvh) [reply src/api/support/support.v6.service.js]: created 'postTicket' method in order to keep the 'create' one too

**Comment by antleblanc** [comment]
Seems related to: #224?

Should I have to left the `draft` mode on it?

Feel free to ask. Thanks!

**Comment by frenautvh** [comment]
> Seems related to: #224?
> 
> Should I have to left the `draft` mode on it?
> 
> Feel free to ask. Thanks!

oh i did not see that, do you want to merge your PR or this one ? it's basically the same

**Comment by antleblanc** [comment]
@frenautvh #224 has been opened to a 'Code Review'.

Feel free to pick it or close it regarding your requirements but just pay attention because
you are introducing a `BREAKING CHANGE` here since you are renaming the [`create`](https://github.com/ovh-ux/ovh-api-services/pull/227/files#diff-cb42313ae79cd660b5bc659162fc7589R24) action URL.

**Pull request closed without merging**